.github/workflows/packages-bin.yml

```diff
@@ -247,8 +258,8 @@ jobs:
             PSYCOPG_IMPL=binary
             PATH="C:\\Program Files\\PostgreSQL\\14\\bin;$PATH"
```

Comment:
This entry can be removed. It might be actually the thing that ends up importing the wrong libpq.